controllers/userController.js

```diff
@@ -0,0 +1,30 @@
+import { UserService } from '../services';
+
+const logIn = async (req, res) => {
+  try {
+    const { user_account, password } = req.body;
+    let err;
+
+    if (!user_account && !password) {
+      err = new Error('PLEASE_ENTER_YOUR_USER_ID_AND_PASSWORD.');
+      err.statusCode = 404;
+      throw err;
+    } else if (!user_account) {
+      err = new Error('PLEASE_ENTER_YOUR_USER_ID.');
+      err.statusCode = 404;
+      throw err;
+    } else if (!password) {
+      err = new Error('PLEASE_ENTER_YOUR_PASSWORD.');
+      err.statusCode = 404;
+      throw err;
+    } else {
+      const access_token = await UserService.logIn(user_account, password);
```

Comment:
```suggestion

    if (!user_account && !password) {
      err = new Error('PLEASE_ENTER_YOUR_USER_ID_AND_PASSWORD.');
      err.statusCode = 400;
      throw err;
    } else if (!user_account) {
      err = new Error('PLEASE_ENTER_YOUR_USER_ID.');
      err.statusCode = 400;
      throw err;
    } else if (!password) {
      err = new Error('PLEASE_ENTER_YOUR_PASSWORD.');
      err.statusCode = 400;
      throw err;
    } else {
      const access_token = await UserService.logIn(user_account, password);
```

`404`는 백엔드에서 해당하는 리소스가 없을 때를 알려주는 에러코드로 지금과 같이 frontend의 input에 문제가 있을 때는
`400 Bad Request`가 보다 적절합니다 :)